rest-service/manager_rest/rest/resources_v3_1/agents.py

```diff
@@ -176,8 +176,13 @@ def put(self, name):
             new_agent = get_storage_manager().get(models.Agent, name)
 
         if request_dict.get('create_rabbitmq_user'):
-            # Create rabbitmq user
-            self._get_amqp_manager().create_agent_user(new_agent)
+            try:
+                # Create rabbitmq user
+                self._get_amqp_manager().create_agent_user(new_agent)
+            except manager_exceptions.ConflictError:
+                # Assuming the agent user was already created
+                current_app.logger.info(f'Not creating agent user "{name}" '
```

Comment:
This would probably be better as a 'Not creating agent user  "%s"', name to avoid wasting (a tiny amount of) time when not hitting the conflict path.